test/framework/controller/installation_manager.go

```diff
@@ -81,6 +84,7 @@ func (m *defaultInstallationManager) computeDefaultHelmVals() map[string]interfa
 		"create": false,
 		"name":   m.controllerSAName,
 	}
+	vals["enableEndpointSlices"] = false
```

Comment:
do we need this?

Reply:
Removed this line, we don't need it since the default value of enableEndpointSlices is false. Thanks for pointing out.